rust/vapid/src/lib.rs

```diff
@@ -24,116 +26,107 @@
 //!
 //! // The result will contain the `Authorization:` header. How you inject this into your
 //! // request is left as an exercise.
-//! let authorization_header = sign(key, &mut claims)?;
+//! let authorization_header = sign(my_key, &mut claims).unwrap();
 //!
 //! ```
 
 extern crate base64;
+#[macro_use()]
```

Comment:
nit: we don't need extern directives nor macro_use's any longer in 2018 edition, prefer using the use statement for the latter instead (not sure I see any openssl macro being used either?)

Reply:
if you haven't already, might wanna run cargo fix in case it picks up anything else as well. IIRC it didn't handle the externs/macro_use's. when I did autopush-rs I did it in two steps:

`cargo fix --edition` then `cargo fix --edition-idioms`